packages/@aws-cdk/aws-redshift/lib/cluster-ref.ts

```diff
@@ -0,0 +1,52 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
+import { IResource } from '@aws-cdk/core';
+import { Endpoint } from './endpoint';
+
+/**
+ * Create a Redshift Cluster with a given number of nodes.
+ */
+export interface ICluster extends IResource, ec2.IConnectable, secretsmanager.ISecretAttachmentTarget {
+  /**
+   * Name of the cluster
+   * @attribute ClusterName
+   */
+  readonly clusterName: string;
+
+  /**
+   * The endpoint to use for read/write operations
+   * @attribute EndpointAddress,EndpointPort
+   */
+  readonly clusterEndpoint: Endpoint;
+
+  /**
+   * The security group for this database cluster
+   */
+  readonly securityGroupIds: string[];
```

Comment:
Is this really needed? `ICluster` extends `ec2.IConnectable`, I think that should be enough?